src/kemal.cr

```diff
@@ -17,13 +20,12 @@ at_exit do
   config.add_handler Kemal::Handler::INSTANCE
   config.add_handler HTTP::StaticFileHandler.new("./public")
 
-  server = HTTP::Server.new("0.0.0.0", config.port, config.handlers)
```

Comment:
Oh... I just checked `kemal.cr` on `master` branch and that `0.0.0.0` line is not there as well.